src/makielayout/blocks/polaraxis.jl

```diff
@@ -273,14 +273,9 @@ function draw_axis!(po::PolarAxis, axis_radius)
         color = po.thetaticklabelcolor,
         strokewidth = po.thetaticklabelstrokewidth,
         strokecolor = thetastrokecolor,
-        align = thetatick_align[]
+        align = thetatick_align,
```

Comment:
If I change this to `thetatick_align[]` again, the StackOverflow goes away. Should we go back to that, or is the problem deeper?

Reply:
Yea this stuff shouldn't get reverted. Git got confused somehow and is doing weird things